packages/file_selector/file_selector/lib/file_selector.dart

```diff
@@ -10,6 +10,18 @@ export 'package:file_selector_platform_interface/file_selector_platform_interfac
     show XFile, XTypeGroup;
 
 /// Open file dialog for loading files and return a file path
+///
+/// The [acceptedTypeGroups] argument is the file type that can be selected in the dialog.
```

Comment:
It's not a file type, it's a list of possible file types.

It would also be useful to give examples of how this will work on different platforms. E.g., on Windows each group will be an entry in a list of filter options, while on macOS (where there is no such UI, the union of all types allowed by all of the groups will be allowed.

Reply:
This has not been resolved.